isodatetime/__init__.py

```diff
@@ -17,4 +17,4 @@
 # ----------------------------------------------------------------------------
 """Python ISO 8601 date time parser and data model/manipulation utilities."""
 
-__version__ = "2018.11.0"
+__version__ = "1!2.0b1"
```

Comment:
Wasn't expecting to find this change snuck in. Why change the version format?

Reply:
Also it would be good to add an `isodatetime --version` option.

Reply:
This has been discussed on chat.

In short:
* Semantic version scheme is preferred by Python community.
* Indeed, our date-based scheme has build issues.
* In order to move to semantic scheme - we need to add a version epoch: https://www.python.org/dev/peps/pep-0440/#version-epochs

Reply:
OK, if I understand the epoch is needed in the `setup.py` for version sorting. Can we omit it from `__version__`:

*`isodatetime/__init__.py`*
```python
__version__ = '2.0'
```

*`setup.py`*
```python
setup(
    name='isodatetime',
    version='1!%s' % version
```

It's just that `1!2.0b1` isn't a very snappy or recognisable version name, the epoch isn't intended for public consumption.

Reply:
Clever. I'll have a go.